initramfs-init.in

```diff
@@ -364,6 +364,11 @@ if [ "${s390x_net%%,*}" = "qeth_l2" ]; then
 	echo 1 > /sys/bus/ccwgroup/drivers/qeth/"${_channel%%,*}"/online
 fi
 
+if [ -n "$ssh_install" ]; then
+	echo "nameserver $nameserver" > /etc/resolv.conf
+	ALPINE_REPO=${KOPT_alpine_repo:-http://dl-cdn.alpinelinux.org/alpine/latest-stable/main}
```

Comment:
This is will not work if someone has `alpine_repo=auto` at the command line. We should probably do something like:
````sh
if [ -n "$ssh_key" ] || [ -n "$ssh_pass" ]; then
    : ${ALPINE_REPO:=http://dl-cdn.alpinelinux.org/alpine/latest-stable/main}
fi
````
`ALPINE_REPO` will be empty if user set `alpine_repo=auto` boot option.

Reply:
This is helpful. Fixed.